src/mcts/search.h

```diff
@@ -152,6 +159,7 @@ class Search {
   const int64_t initial_visits_;
 
   mutable SharedMutex nodes_mutex_;
+  std::list<EdgeAndNode> asc_move_edges_ GUARDED_BY(nodes_mutex_);
```

Comment:
Not sure this needs to be a member variable? Can't it just be a local variable in like the two places it's needed?

Reply:
I really wasn't sure if I was going to cause a bug with multi-threading when I made this decision but I see that SendUciInfo requires the nodes_mutex_ so on second thought it shouldn't be modified in that function.